sdk/web-pubsub/web-pubsub-client/src/webPubSubClient.ts

```diff
@@ -709,6 +736,13 @@ export class WebPubSubClient {
           this._safeEmitServerMessage(message);
         };
 
+        const handlePongMessage = (): void => {
```

Comment:
Considering the pong is only useful when there's no other messages coming, we can think it's still active when the client receives any messages